src/Protocol/Diaspora.php

```diff
@@ -2308,16 +2310,16 @@ private static function receiveRequestMakeFriend($importer, $contact)
 				$A = "[url=".$self[0]["url"]."]".$self[0]["name"]."[/url]";
 				$B = "[url=".$contact["url"]."]".$contact["name"]."[/url]";
 				$BPhoto = "[url=".$contact["url"]."][img]".$contact["thumb"]."[/img][/url]";
-				$arr["body"] = sprintf(t("%1$s is now friends with %2$s"), $A, $B)."\n\n\n".$Bphoto;
+				$arr["body"] = sprintf(t('%1$s is now friends with %2$s'), $A, $B)."\n\n\n".$BPhoto;
 
 				$arr["object"] = self::constructNewFriendObject($contact);
 
 				$arr["last-child"] = 1;
 
-				$arr["allow_cid"] = $user[0]["allow_cid"];
-				$arr["allow_gid"] = $user[0]["allow_gid"];
-				$arr["deny_cid"]  = $user[0]["deny_cid"];
-				$arr["deny_gid"]  = $user[0]["deny_gid"];
+				$arr["allow_cid"] = $self[0]["allow_cid"];
```

Comment:
```$user``` is undefined - but ```$self``` doesn't seem to be an appropriate replacement. It should be the defaults for posts that are defined in the ```user``` table. Please add a call for the ```user``` entry here.